ecs-demo/README.md

```diff
@@ -1,14 +1,13 @@
 # KEDA Demo Setup
```

Comment:
Lol I already merged the PR, but just noticed that the title is "KEDA Demo Setup".  Shouldn't it be "ECS Scaler Demo" or something?